sum.tex

```diff
@@ -0,0 +1,18 @@
+\section{High level summary}\label{sec:sum}
+Though the reason behind the call for security and the level required remain totally unclear
+NSF and DOE asked for a brief summary of possibilities.
+One should also consider which data we are talking about see \secref{sec:which}.
+Items here are not costed but an indication is given in terms of low (possibly within cost), moderate (some \$100Ks), high (>\$1M)
+Any change should be properly costed.
+
+
+\begin{longtable}{p{0.75\textwidth} p{0.25\textwidth}}\hline
+\textbf{Security idea} & \textbf{Rough cost level}  \\\hline
+ {\bf Delay/degrade image info.} If the precise position of small objects is the driver then not providing accurate shutter times would make precise positions inaccessible & Low to Moderate.\\
+ {\bf Delay some or all images.} Depending on how secure this needs to be and where it is done the cost scales.  & Low to Moderate.\\
+ {\bf Do alerts in Chile.} If we want to control image access for a longer period we could consider alert production in Chile. The hardware budget would remain the same but we may require extra support in Chile.  & Low to Moderate.\\
+{\bf Encrypt all images.} This would have to be done on the summit or in la Serena before hitting the long haul network. Possibly no new hardware needed but a change in software.  & Can be Low \\
```

Comment:
This summary looks good to me, but elsewhere in `body.tex` there's a statement that encryption would potentially double the number of required cores. That should probably be made consistent with this bullet point, which I think is more likely. Given that we're doing image compression, the additional compute cost for image encryption should be minimal. The primary cost would be the work of switching from `bbcp` to something based on TLS (software changes at both ends) and tuning that for speed.

Personally, I would avoid suggesting WireGuard (also currently mentioned in `body.tex` -- GitHub for some reason won't let me comment on that) since this path would create other challenges. WireGuard has only now been merged into the Linux kernel and is not yet in a stable Ubuntu release let alone the slower-moving releases based on Red Hat Enterprise. It will require some shakeout to catch bugs; I would be hesitant to use it for something this mission-critical for at least another three years.

There's also discussion there about encrypting the images outside of the data transfer. I'm pretty sure just using TLS for the transfer will be simpler and faster all around *unless* there is also some concern about the images at rest in Chile before the transfer.

Reply:
ok changed the compression in text to be may not need increase, put caveat on wireguard that its not ready for us yet. Agree on TLS but they need to tell us.